template.yaml

```diff
@@ -91,7 +91,7 @@ Resources:
     Properties:
       CodeUri:
         Bucket: !Ref DistBucket
-        Key: !Sub ${Stack}/${Stage}/set-retention/lambda.zip
+        Key: !Sub ${Stack}/${Stage}/set-retention/cloudwatch-logs-management.zip
```

Comment:
`@guardian/node-riffraff-artifact` (sensibly IMO!) will always name a file `action/action.zip` as described in the  `artifact.json` file.

Here (and below) we bring the CFN inline with this naming.